guides/v2.1/contributor-guide/backward-compatible-development/index.md

```diff
@@ -434,3 +434,8 @@ In order to update the page, create a PR to the DevDocs repository with your cha
 [versioning]: {{ page.baseurl }}/extension-dev-guide/versioning/index.html
 [devdocs-repo]: https://github.com/magento/devdocs
 [2.2-bic-page]: https://github.com/magento/devdocs/blob/develop/guides/v2.2/release-notes/backward-incompatible-changes/index.md
+
+## GraphQl backward compatibility policy
```

Comment:
Hi @vpodorozh 
Thanks for your contribution.
Can you please move this code to line number 433. because [versioning] is a variable and Devdocs in variable defined at the bottom of the page.

Reply:
Thx, yeh - you are right, my fault!
Also I do see that I have contributed into the 2.1v - but it meant to be done for 2.3v :)
Will fix it today evening.

Thx!

Reply:
@ronak2ram - I've made requested changes.
However, I have the other question - `GraphQl backward compatibility policy` should be added only for v2.3, but the current article is hosted in v2.1 and just symlink to v2.3.
Should I remove this symlink and copy&paste this article in v2.3 to made changes only there? (it does not sound good because we will need to maintain couple versions of this article).

Thx!

Reply:
Hi @vpodorozh you are correct. You have to delete symlink file for v2.3. And just copy file v2.1 and paste in v2.3 the do require PR changes.

Reply:
@ronak2ram - finally done.
Thx!